prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -511,6 +511,56 @@ ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_pee
 	return 0;
 }
 
+ssize_t rxr_pkt_trigger_handshake(struct rxr_ep *ep,
+				  fi_addr_t addr, struct rxr_peer *peer)
+{
+	struct rxr_tx_entry *tx_entry;
+	ssize_t err;
+
+	if (peer->flags & RXR_PEER_HANDSHAKE_RECEIVED)
+		return 0;
```

Comment:
also check ```peer->flags``` with ```RXR_PEER_REQ_SENT```. If it is true, there is no need to send a rtw again.

Reply:
Check added. Resolving.